### PR TITLE
Tw process success

### PR DESCRIPTION
**Pull request by StevePole**

### CODE REVIEW

File: docs/_includes/components/process.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
+<div class='bs-docs-section' id='process'>
+  <h1 id='process-basic' class='page-header'>Process</h1>
+
+  <p>Process is used for loading and processing states. It can be used as a simple
+    loading indicator, for asynchronous tasks with positive or negative results,
+    or simply for indicating a task has already completed with successful or
+    unsuccessful result.</p>
+
+  <div class='alert alert-warning'>
+      <h4>Please don't use SVG the markup directly</h4>
```
Comment (adrienDog) [inline docs/_includes/components/process.html]: --> "Please don't use **the** SVG ~the~ markup directly"?

File: less/process.less [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+@process-lg-circumference: (pi() * (2 * (@process-lg-radius - 1)));
+@process-lg-icon-size: 48px;
+@process-lg-icon-stroke: 6px;
+
+@process-md-radius: 36px;
+@process-md-circumference: (pi() * (2 * (@process-md-radius - 1)));
+@process-md-stroke: 2.5px;
+@process-md-icon-size: 36px;
+@process-md-icon-stroke: 4px;
+*/
```
Comment (adrienDog) [inline less/process.less:19]: Why is this all commented out?

**Review by adrienDog — APPROVED** [review]
I left almost useless comments ^^ that's why I approve directly

File: less/process.less (hunk 2 above)
Comment (StevePole) [reply less/process.less:19]: I originally made variants at 5 different sizes, we don't currently have use cases for the others, so I removed them from the code, but I left the variables as tuning the stroke widths was a little tricky, and they may well come back.

File: docs/_includes/components/process.html (hunk 1 above)
Comment (StevePole) [reply docs/_includes/components/process.html]: Good catch, will update.

**Pull request merged**